ext/opentelemetry-ext-grpc/tests/test_server_interceptor.py

```diff
@@ -49,6 +53,43 @@ def service(self, handler_call_details):
 
 
 class TestOpenTelemetryServerInterceptor(TestBase):
+    def test_instrumentor(self):
+        def handler(request, context):
+            return b""
+
+        grpc_server_instrumentor = GrpcInstrumentorServer()
+        grpc_server_instrumentor.instrument()
+
+        server = grpc.server(
+            futures.ThreadPoolExecutor(max_workers=1),
+            options=(("grpc.so_reuseport", 0),),
+        )
+
+        server = grpc_server_instrumentor.server
```

Comment:
Unnecessary if you return server from `grpc.server()`, it should already be instrumented

Reply:
For the tests and some of the implementations we need a reference to the server. However inorder to have a reference to server we would need to return the server in the wrapper_fn and then in the _instrument function as well, which does not seem to be the standard across other instrument methods.

Reply:
You already got the instrumented server in the lines
```
server = grpc.server(
            futures.ThreadPoolExecutor(max_workers=1),
            options=(("grpc.so_reuseport", 0),),
        )
```
why do you need to do
```
server = grpc_server_instrumentor.server
```
is my question

Reply:
ah I think I see what your saying